auto-prune.sh

```diff
@@ -1,36 +1,86 @@
 #!/bin/bash
-percent_threshold=10
-byte_threshold=104857600
+__percent_threshold=10
+__byte_threshold=104857600
 
-if [ "$(dpkg-query -W -f='${Status}' bc 2>/dev/null | grep -c "ok installed")" = "0" ]; then
-  echo "This script requires the bc package, please install it via 'sudo apt install bc'"
+if [ "$(dpkg-query -W -f='${Status}' bc 2>/dev/null | grep -c "ok installed")" = "0" ] || [ "$(dpkg-query -W -f='${Status}' jq 2>/dev/null | grep -c "ok installed")" = "0" ]; then
+  echo "This script requires the bc and jq packages, please install them via 'sudo apt install bc jq'"
   exit 1
 fi
 
 cd "$(dirname "$0")" || exit 1
 
-dryrun=0
-for var in "$@"; do
+
+__docker_dir="/var/lib/docker"
+if [ -f "/etc/docker/daemon.json" ]; then
+  __dir=$(jq '."data-root"' /etc/docker/daemon.json)
```

Comment:
Thanks for the change

We need to make sure we use the `--raw-output` flag for jq, otherwise it will include the double-quotes in the env variable's value itself.

```suggestion
  __dir=$(jq --raw-output '."data-root"' /etc/docker/daemon.json)
```

This change works well in my case, but it will fail for folks who manually change their `data-dir` using the `-g` flag in their systemctl launch config (rather than using `/etc/docker/daemon.json`). Pretty minor issue though and probably not high-pri.

Reply:
Thanks for the suggestion! Change made. I am not too worried about users of `-g`.